<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>filter_gui</class>
 <widget class="QWidget" name="filter_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>过滤器</string>
  </property>
  <widget class="QCheckBox" name="checkBox_sip">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>源地址</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_dip">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>目的地址</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_port">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_protocal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>协议</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Any Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_dip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Any Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Any Protocal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_protocal">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>tcp or udp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>filter_gui</receiver>
   <slot>setFilter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setFilter()</slot>
 </slots>
</ui>
